.github/workflows/python-package.yml

```diff
@@ -0,0 +1,30 @@
+name: Python package
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
```

Comment:
consider also adding cron runs here

Reply:
Example: https://github.com/pypa/setuptools/blob/df9157c/.github/workflows/python-tests.yml#L8-L9

Reply:
```suggestion
    branches: [main]
  schedule:
  - cron: 1 0 * * *  # Run daily at 0:01 UTC
```

Reply:
Ah, would this be to catch nightly updates?

Reply:
Mostly to catch any env/unpinned deps updates

Reply:
But yeah, you can also hit Python nightlies but that would probably require using the deadsnakes action in addition to GHA-shipped CPython builds.

Reply:
re:deadsnakes -- here's a canonical example: https://github.com/deadsnakes/action#using-this-action